pkg/crd/crd.go

```diff
@@ -0,0 +1,136 @@
+// Package crd helps with the installation of CustomResourceDefinitions.
```

Comment:
Tiny nit: I'd add a more descriptive or concrete comment, not sure what "helps" really means.